{"name": "test_home_elementos_autenticado[chrome]", "status": "failed", "statusDetails": {"message": "AssertionError: Should be redirected to dashboard after login\nassert 'dashboard' in 'http://web:8000/accounts/login/'\n +  where 'http://web:8000/accounts/login/' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"31fd9141357cf8297cbe7a9062d2670b\")>.current_url", "trace": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/runner.py\", line 262, in <lambda>\n    lambda: ihook(item=item, **kwds), when=when, reraise=reraise\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/runner.py\", line 177, in pytest_runtest_call\n    raise e\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/runner.py\", line 169, in pytest_runtest_call\n    item.runtest()\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/python.py\", line 1792, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/python.py\", line 194, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/app/tests/home_autenticado_test.py\", line 27, in test_home_elementos_autenticado\n    assert \"dashboard\" in driver.current_url, \"Should be redirected to dashboard after login\"\nAssertionError: Should be redirected to dashboard after login\nassert 'dashboard' in 'http://web:8000/accounts/login/'\n +  where 'http://web:8000/accounts/login/' = <selenium.webdriver.remote.webdriver.WebDriver (session=\"31fd9141357cf8297cbe7a9062d2670b\")>.current_url"}, "description": "TP-16: Home - Elementos para Usu√°rio Autenticado", "steps": [{"name": "Make request to Faker API: http://faker:5000/generate/user", "status": "passed", "start": 1740707456414, "stop": 1740707456418}], "attachments": [{"name": "log", "source": "55af883a-f2e6-43fb-ae7c-88acffa7744b-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "ee5a2850-c3f1-4649-b1b5-646c21a9118e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "selenium_browser", "value": "'chrome'"}], "start": 1740707456414, "stop": 1740707456983, "uuid": "f01245fa-a5e0-4e67-8259-48b9f31c74ee", "historyId": "26a5b6c68cbbc653639b5194410a3aca", "testCaseId": "1eebfb6de96b4b529e9f8d8c593dce09", "fullName": "tests.home_autenticado_test#test_home_elementos_autenticado", "labels": [{"name": "story", "value": "Authenticated View"}, {"name": "feature", "value": "Home Page"}, {"name": "epic", "value": "User Interface"}, {"name": "tag", "value": "ui"}, {"name": "tag", "value": "nondestructive"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "home_autenticado_test"}, {"name": "host", "value": "e130d6c28101"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.home_autenticado_test"}]}